rust-tests/src/lib.rs

```diff
@@ -629,4 +629,24 @@ mod tests {
 
         assert!(rattler_build_render_only.status.success());
     }
+
+    #[test]
+    #[cfg(target_os = "macos")]
```

Comment:
Only macos? :)

Reply:
currently this issue can only be reproduced on macos when using --target-platform win-64 ( because macos has insensitive file system ) - so I've decided to keep this test to this only scope